Build/run.sh

```diff
@@ -64,7 +64,7 @@ then
   /usr/bin/docker run --privileged ${USE_TTY} --rm -e NO=${NO} -e SDK=${SDK} -e DLOAD=${DLOAD} -v "${PWD}/${OUTDIR}":/home/yocto/rzg_vlp_v3.0.0/out --name ${CONTNAME} ${CONTNAME}
 else
 	#Create CPATH sub directories if they do not exist
-	mkdir -p ${CPATH}/downloads
-	mkdir -p ${CPATH}/sstate-cache/${MPU}
+	mkdir -p -m777 ${CPATH}/downloads
+	mkdir -p -m777 ${CPATH}/sstate-cache/${MPU}
 	/usr/bin/docker run --privileged ${USE_TTY} --rm -v "${PWD}/${OUTDIR}":/home/yocto/rzg_vlp_v3.0.0/out -v "${CPATH}/downloads":/home/yocto/rzg_vlp_v3.0.0/build/downloads -v "${CPATH}/sstate-cache/${MPU}/":/home/yocto/rzg_vlp_v3.0.0/build/sstate-cache -e NO=${NO} -e SDK=${SDK} -e DLOAD=${DLOAD} --name ${CONTNAME} ${CONTNAME}
```

Comment:
Should probably remove all references to ${DLOAD} in this file

Reply:
was just a draft but i've cleaned it up now!